tabular/tests/unittests/models/test_tabular_nn.py

```diff
@@ -51,6 +51,19 @@ def test_tabular_nn_binary_compile_onnx(fit_helper):
     assert isinstance(predictor._learner.trainer.models["NeuralNetTorch"].processor, TabularNeuralNetTorchOnnxTransformer)
 
 
+def test_tabular_nn_binary_compile_onnx_as_ag_arg(fit_helper):
+    fit_args = dict(
+        hyperparameters={TabularNeuralNetTorchModel: {"ag.compile": {"compiler": "onnx"}}},
+    )
+    dataset_name = "adult"
+    predictor = fit_helper.fit_and_validate_dataset(dataset_name=dataset_name, fit_args=fit_args, refit_full=True, delete_directory=False)
+    from autogluon.tabular.models.tabular_nn.compilers.onnx import TabularNeuralNetTorchOnnxTransformer
+
+    assert isinstance(predictor._learner.trainer.load_model("NeuralNetTorch").processor, TabularNeuralNetTorchOnnxTransformer)
```

Comment:
QQ: Just wanted to understand the difference here between `NeuralNetTorch` and `NeuralNetTorch_FULL`

Reply:
NeuralNetTorch is the original model fit with train and validation data.

NeuralNetTorch_FULL is the refit model (via `refit_full=True`) which combines the train and validation data together into a combined train data and fits the model again. The unit test checks that compilation works correctly for both the original and _FULL model.